vpr/src/place/cut_spreader.cpp

```diff
@@ -0,0 +1,816 @@
+#include "cut_spreader.h"
```

Comment:
Needs a top-level comment that explains what this file does, with an overview of the algorithm, references to HEAP / SimPL etc.
Subroutines also need comment as to what each does.